src/oss/confluent.sh

```diff
@@ -1741,6 +1741,11 @@ invalid_requirement() {
 
 requirements
 
+echo "WARNING: This Command Line tool is not intended for production use.
+For production environment documentation, please refer to:
+https://docs.confluent.io/current/installation/installing_cp.html
+"
```

Comment:
it looks like the rest of the file uses the `cat <<EOF` (example below) to print multiline strings. maybe we should be consistent?
```
    cat <<EOF
Usage: ${command_name} list [ plugins | connectors ]
Description:
    List all the available services or plugins.
    Without arguments it prints the list of all the available services.
...
```